R/report_htest_ttest.R

```diff
@@ -78,3 +78,38 @@
     rules = rules, ci = ci, main = main
   )
 }
+
+
+# report model ---------------------------
+
+.report_model_ttest <- function(x, table) {
+  # If against mu
+  if (names(x$null.value) == "mean") {
+
+    # TODO: @DominiqueMakowski why do we need "table" here?
```

Comment:
ping @DominiqueMakowski

Reply:
I think it was for standardization ? can't remember though